app/scripts/views/settings/two_step_authentication.js

```diff
@@ -66,6 +67,11 @@ const View = FormView.extend({
     if (this.broker.hasCapability('showTwoStepAuthentication')) {
```

Comment:
I was wrong, that'll cause a failure in the functional tests because experiments are disabled for functional tests unless explicitly adding the `forceExperiment` and `forceExperimentGroup` query params.